scripts/ci/VM-container-tests.sh

```diff
@@ -182,9 +218,10 @@ qemu-system-x86_64 -machine accel=kvm,vmport=off -m 1024G -smp 4 -cpu host -bios
   # -serial mon:stdio -display curses
 
 # Waiting for VM's ssh server to start and get cert
-sleep 1
+sleep 10
 echo "STATUS: Waiting for VM ssh server to start up..."
```

Comment:
We should remove stale ssh keys from previous test runs to avoid that the log gets spammed with unnecessary warnings.

ssh-keygen -f "$HOME/.ssh/known_hosts" -R "[localhost]:$SSH_PORT"